compilation/src/main/scala/scala/tools/nsc/ScalacBenchmark.scala

```diff
@@ -43,6 +46,12 @@ class ScalacBenchmark extends BenchmarkDriver {
   @Param(value = Array())
   var scalaVersion: String = _
 
+  var scalaJars: String = _
+
+  override def findScalaJars: String =
+    if (scalaJars != null) scalaJars
+    else super.findScalaJars
```

Comment:
The build is failing because of this line. How do I make this compile in `dotty`?

Reply:
there are some scalac/dotc specific sources under `src/main/{dotc,scalac}` which might need to be changed if you change this.

Reply:
You can use `sbt ++0.2.0-RC1 compilation/test` to test against dotty.